aptos-move/aptos-vm-environment/src/prod_configs.rs

```diff
@@ -127,12 +129,15 @@ pub fn aptos_prod_vm_config(
         256
     };
 
+    // Value runtime depth checks are enabled only when function values are enabled.
+    let enable_depth_checks = features.is_enabled(FeatureFlag::ENABLE_FUNCTION_VALUES);
```

Comment:
The comment could be more descriptive, like "Value runtime depth checks have been introduced together with function values and are only enabled when ...", otherwise someone will wonder later why this.

Reply:
Changed